indico/modules/events/registration/templates/management/_registration_details.html

```diff
@@ -91,7 +91,9 @@
             <a class="i-button accept"
                data-update="#registration-details"
                data-method="POST"
-               data-href="{{ url_for('.approve_registration', registration) }}">
+               data-href="{{ url_for('.approve_registration', registration) }}"
+               data-confirm="{% trans %}Are you sure you want to approve this registration? This will trigger a notification email.{% endtrans %}"
+               data-title="{% trans %}Approve registration{% endtrans %}">
```

Comment:
Is a confirmation prompt for what's probably the most common action really needed, especially now that it can be undone in case someone does misclick?

Reply:
I think it's generally a good idea to let managers know that an action will trigger an email. But now, precisely because approvals can be undone, there is the risk for managers to inadvertently triggering multiple emails. There could be better ways to hint that an email will be sent, but I think this is better than nothing. At the same time, I don't have a very strong opinion on this. Let me know if I should remove it.

Reply:
I don't have a very strong opinion either. But then it would be logical that we add this warning also when we bulk-approve registrations.

Reply:
Reworded the already-existing confirmation dialog for the bulk-moderation actions. 👌